packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -140,7 +140,7 @@ export class HorizontalScroll extends FoundationElement {
      * Attribute to hide flippers from assistive technology
      * @public
      */
-    @attr({ attribute: "aria-hidden", converter: booleanConverter })
+    @attr({ attribute: "hide-flippers", converter: booleanConverter })
```

Comment:
Only suggestion here would be to specify that this is for AT (assistive tech). This implies a bit that it also visually hides it, when ideally it wouldn’t prevent render and they’d still show (on hover only technically…)

Reply:
I went back and forth on this and thought the same thing. But I was torn with using a really verbose attribute `flippers-hidden-from-at` and this. I figured that the way to hide flippers would be to slot in an empty element or through CSS so maybe it would be alright to be less explicit here. But it makes sense.